### PR TITLE
Add YBe Generator

### PR DESCRIPTION
**Pull request by ghusheng**
Add `generator_ybe` in the generator.py , the default YBe instruction file path is `/project2/lgrandi/ghusheng/ybe_instrutions/ybe_wfsim_instructions_6806_events_time_modified.csv`

### CODE REVIEW

**Comment by ghusheng** [comment]
Maybe to combine ambe and ybe generator (also maybe neutron_gun in the future) to nr_generator?

File: .singularity_inner
Comment (FaroutYLq) [inline .singularity_inner:1]: This should go

File: jobs/config.ini
Comment (FaroutYLq) [inline jobs/config.ini:1]: I would keep the template so not to touch this

File: jobs/config_ybe.ini
Comment (FaroutYLq) [inline jobs/config_ybe.ini:1]: I would keep the original config as a template

File: saltax/instructions/.singularity_inner
Comment (FaroutYLq) [inline saltax/instructions/.singularity_inner:1]: This should go

File: saltax/instructions/generator.py
Comment (FaroutYLq) [inline saltax/instructions/generator.py:1]: It looks good to me, but I am a bit lazy to test, so I would just trust you:)

**Review by FaroutYLq — CHANGES_REQUESTED** [review]
Thanks a lot @ghusheng. I would just keep the generator itself and remove other things. One thing we might need to think about is to put all the csv inputs (together with the AmBe's) into a more shared-like folder something like: `/project/lgrandi/xenonnt/saltax_inputs`. Maybe in another PR. 
Meanwhile, since AmBe and YBe are so different in energy and positions I would avoid putting them into one generator. It makes no sense to me to sprinkle ambe neutrons into a ybe runs.

File: saltax/instructions/generator.py
Comment (ghusheng) [reply saltax/instructions/generator.py:1]: let me do a minor change

**Comment by FaroutYLq** [comment]
Ignore the tests, they are nonsense for now.

**Comment by ghusheng** [comment]
should be all good now

**Pull request merged**